printer_mods/README.md

```diff
@@ -146,6 +146,7 @@ like so:
 | | [Mid Panel](./JNP/Mid-Panel)| Panel for SKR1.4/1.4 or SKR E3 mini mount | :heavy_check_mark: |:x: |:x: |:x: |
 | | [Neopixel Case](./JNP/Neopixel-Y_Rails) | mounting for Neopixel strip with 8 LED´s | :heavy_check_mark: |:x: |:x: |:x: |
 | | [Side Panels](./JNP/Side-Panels) | Panels with intergrated hinge | :heavy_check_mark: |:x: |:x: |:x: |
+| Jon | [adxl345_skirt_keystone](./jon/adxl345_skirt_keystone) | Skirt keystone for ADXL345 using Microfit 3.0 | :x: | :x: | :heavy_check_mark: | :grey_question: | :grey_question: | :heavy_check_mark: |
```

Comment:
Please change the text in square brackets to remove underscores (this is what's displayed to the user), and also fix your link, your directory is called `Jon` but your link points to `jon` (case matters).  It should be

    [adxl345 skirt keystone](./Jon/adxl345_skirt_keystone)